eclipse-theia/theia
# repo: upgrade `mocha`

#### What it does


The commit includes the following changes:
- upgrades `lerna`
- upgrades `mocha` including our config files
- upgrades `minimatch`

The main reason of the upgrade was to fix known vulnerabilties and benefit from the newer mocha version.

#### How to test


- CI for unit-tests, and the integration tests should pass
- running individual tests (ex: `npx run test @theia/core`) should work
- running individual tests using the `run mocha tests` launch configuration should work
- `yarn audit` should not produce vulnerabilities (`jsdom` is expected)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
